### DISCUSSION TITLE
Not getting approved

### DISCUSSION THREAD

**Discussion by Roxen-AS** [community/community#135044] [GitHub Education]
### Hi everyone,

I have been trying to submit my application for github student developer pack and although i have valid id proof and documents, since they are in the form of pdfs and images i am unable to upload them and when i try clicking apic of them to upload it doesn't get approved. Also, I am away from my campus as I have a semester break and the next semester will start a little late. Can someone help me out with what to do and how to proceed with this please?

**Comment by ahmedembeddedxx** [comment]
First of all, there's an option to upload a picture, which mostly gets disapproved. You can try uploading screenshots of the PDF; if that works out, great. 

It's better to have your original documents. If that's not possible, you can take high-quality images of the PDFs with your device camera. They generally approve applications when you provide pictures taken directly from the base devices. Just make sure the picture is clear and taken using a live camera.
